pyccel/ast/core.py

```diff
@@ -3167,6 +3168,7 @@ def local_vars(self):
 
     @property
     def global_vars(self):
+        sys.exit("global vars in functiondef\n")
         return self._global_vars
```

Comment:
we need to keep this one, in order to have access to the global variables used in the function